ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -298,8 +363,30 @@ open class SegmentedControl: UIControl {
         var rightOffset: CGFloat = 0
         var leftOffset: CGFloat = 0
         for (index, button) in buttons.enumerated() {
-            if style.segmentsHaveEqualWidth {
-                rightOffset = UIScreen.main.roundToDevicePixels(CGFloat(index + 1) / CGFloat(buttons.count) * frame.width)
+            if segmentsHaveEqualWidth {
+                if (style == .tabs)
+                {
+                    rightOffset = UIScreen.main.roundToDevicePixels(CGFloat(index + 1) / CGFloat(buttons.count) * frame.width)
+                } else {
+                    var suggestedWidth: CGFloat = frame.width
+                    var availableWidth: CGFloat = suggestedWidth
+
+                    if let windowWidth = window?.frame.width {
+                        suggestedWidth = windowWidth
+                    }
+                    // for iPad regular width size, notification toast might look too wide
+                    if traitCollection.userInterfaceIdiom == .pad &&
+                        traitCollection.horizontalSizeClass == .regular &&
+                        traitCollection.preferredContentSizeCategory < .accessibilityMedium {
```

Comment:
is this needed?

Reply:
My guess is that it's not. It prevents the control from taking up the entire width of the screen.
Good logic for the NotificationView, but I believe it shouldn't be a requirement here:

![Screen Shot 2021-02-26 at 4 23 14 PM](https://user-images.githubusercontent.com/68076145/109368730-ee8e0880-784e-11eb-8e19-52024078dd51.png)

Reply:
Looks like we don't need the second or third parts, but we need the first part for the control to not go all the way across the iPad.

Reply:
Is this an intended behavioral change compared to the deprecated tabs above?
I don't think it makes much sense to limit the buttons but still let the bar take the space with an empty half like below:

![Screen Shot 2021-03-01 at 1 31 52 PM](https://user-images.githubusercontent.com/68076145/109561771-7ca60100-7a92-11eb-91f9-73660e3ae1fc.png)

Reply:
yeah it is a design driven decision. Although hunter we should center align this control in the demo app.